shared/bazel/rules/objectivec_rules.bzl

```diff
@@ -0,0 +1,29 @@
+load("@rules_cc//cc:defs.bzl", "objc_library")
+
+OBJC_COMPILER_FLAGS = [
```

Comment:
This deserves a comment or 3.  Why do we need this?  Why do the upstream rules not handle this already?

Reply:
This was done to match the gradlerio compiler flags. I tried to have everything match when I first stood this up.

Reply:
Can you put that in a comment?  That alone is super helpful :)